### PR TITLE
cmd/bosun: annotate support

### PR DESCRIPTION
**Pull request by kylebrandt**

### CODE REVIEW

**Comment by kylebrandt** [comment]
yiwa6ucoka

File: cmd/bosun/web/web.go [hunk 1]
```diff
@@ -119,6 +124,25 @@ func Listen(listenAddr string, devMode bool, tsdbHost string) error {
...
+			index = "annotate"
+		}
+		annotateBackend, err = backend.NewElastic(schedule.Conf.AnnotateElasticHosts, index)
+		if err != nil {
+			return err
+		}
+		if err := annotateBackend.InitBackend(); err != nil {
+			return err
+		}
+		web.AddRoutes(router, "/api", []backend.Backend{annotateBackend}, false, false)
```
Comment (captncraig) [inline cmd/bosun/web/web.go]: I like this way of doing it a lot. What are the last two bool parameters for? Mostly out of curiosity.

Comment (captncraig) [inline cmd/bosun/web/web.go]: nvm. read the code.

Comment (captncraig) [inline cmd/bosun/web/web.go]: are we gonna try to get the ui working at some point?

Comment (kylebrandt) [inline cmd/bosun/web/web.go]: enable the independent UI, and use local dev files when doing so

Comment (kylebrandt) [inline cmd/bosun/web/web.go]: The UI does work, but I don't intend to just import the UI into bosun, Bosun will integrate with it using it's own UI.

**Comment by captncraig** [comment]
LGTM

File: cmd/bosun/web/web.go (hunk 1 above)
Comment (captncraig) [inline cmd/bosun/web/web.go]: was thinking it might be nice if your ui has direct query/filter/ raw viewing support.

Comment (kylebrandt) [inline cmd/bosun/web/web.go]: Agreed, approaching this incremently and thinking of it as probably a "preview" feature when we release

**Pull request merged**